platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java

```diff
@@ -133,6 +134,9 @@ public HandlerResult getNormalizedToken(
 
     HttpSession session = httpRequest.getSession(false);
     if (session == null && httpRequest.getRequestedSessionId() != null) {
+      if (httpRequest.isRequestedSessionIdValid()) {
+        SecurityLogger.audit("Request has requested an invalid session ID for this server.");
```

Comment:
nit: I wonder if `Incoming HTTP Request contained invalid session ID for this server` might make a better message.

Reply:
Sure